g3doc/index.md

```diff
@@ -1188,6 +1201,8 @@ methods on `NSArray`, which often comes handy, along with other similar
 [functional operators](https://github.com/google/functional-objc) that
 Objective-C lacks.
 
+Note: The number of promises that `any` can handle is 3.
```

Comment:
Can you phrase the comment here similarly to what you had for all?
In Swift, the number of promises of heterogeneous types that `any` can handle is 3.

Reply:
yes I can.
I'll be happy to do it.